docs/kratos/emails-sms/01_sending-emails-smtp.mdx

```diff
@@ -66,12 +71,6 @@ To configure your credentials for the custom SMTP server, go to **Customize** 
 </Tabs>
 ```
 
-:::note
-
-Customizing the `from_address` and `from_name` is not supported without a custom SMTP server.
```

Comment:
I think we should keep this note around in the docs. Could probably use some explaining, but I am not sure what the exact reason is. Something with email security, @aeneasr do you remember what the reason is here, and what I can google to read up on it a bit?

Reply:
Oh, I didn't remove it! Have a look: 
https://github.com/ory/docs/pull/1203/files#diff-0f16158a8aa7b0d79471d75c109ea45b43e9ebd8d33019b707f3bd2dec717d2bR22

Reply:
Found it thanks for the clarification :)